app/controllers/data_lists_controller.rb

```diff
@@ -268,10 +272,26 @@ def remove_measurement
     DataListMeasurement.destroy(id_to_remove)
   end
 
+  def set_measurement_indent
+    dlm = DataListMeasurement.find_by(data_list_id: params[:id], measurement_id: params[:measurement_id])
+    curind = dlm.indent ? dlm.indent[-1].to_i : 0
+    newind = params[:indded] == 'plus' ? curind + 1 : curind - 1
+    if newind < 0 || newind > MAXINDENT
+      respond_to do |format|
+        format.js { render nothing: true, status: 200 }
+      end
+      return
+    end
+    respond_to do |format|
+      format.json { render json: '{ "the_indent": "%s" }' % view_context.make_indentation(newind), status: 200 }
+    end
+    dlm.update(indent: 'indent'+newind.to_s)
+  end
+
   private
     def data_list_params
       params.require(:data_list)
-          .permit(:name, :list, :startyear, :created_by, :updated_by, :owned_by, :measurements, :measurement_id)
+          .permit(:name, :list, :startyear, :created_by, :updated_by, :owned_by, :measurements, :measurement_id, :indded)
```

Comment:
What is `indded`? Can we use more descriptive names? It will make our codebase more maintainable. The same comment applies to `curind`, `newind`. I can live with the abbreviated model name, since we use that pattern liberally throughout the code base, but otherwise we should limit abbreviation. Alternatives in this case could be `current_indent`, `new_indent`. I can't figure out what the long form of `indded` would be.